### ISSUE TITLE
DietPi-Update | Update reporting older version as newer

### ISSUE THREAD

**Issue by DasClarky** [MichaIng/DietPi#1395]
dietpi-update    = Run now to update DietPi (from V159 to V157).

I doubt this is normal and i run updates as instructed.

**Comment by DasClarky** [comment]
I rebooted and it disappeared but still weird

**Comment by k-plan** [comment]
Please have a look at:

https://github.com/Fourdee/DietPi/issues/1258

**Comment by DasClarky** [comment]
Thanks k-plan! Didn't notice that was already an issue

**Comment by k-plan** [comment]
To fix it:

```
rm /DietPi/dietpi/.update_available

rm /boot/dietpi/.update_available

dietpi-update 1


```

**Comment by k-plan** [comment]
@Fourdee 

leave it open, so you can have a look, if additional fix is require:

https://github.com/Fourdee/DietPi/commit/78ca55697fbba380fcb78fa4c650ce3eee455381#diff-c613a85da508fb885b67c34f9661e243R137

```
root@RPi3:~# cat /DietPi/dietpi/.update_available
159

root@RPi3:~# cat /boot/dietpi/.update_available
159

```
Please close, if it is fixed. Thanks.

**Comment by Fourdee** [comment]
@k-plan 

Should be fixed for v159. To clear the existing update available flag, requires a script run of dietpi-update which auto runs during boot, or when user runs ```dietpi-update```.

We did have a possible issue with dietpi-ramdisk not saving to disk during shutdown, this is resolved for v6.0.

I'll mark this as closed for now, unable to replicate it in testing. We can reopen if v6.0 users report the same issue.
